lesson_00/Kenrick/Readme.md

```diff
@@ -39,7 +39,7 @@ I've been to Aruba, Curacao, and the Dominican Republic, enjoying their awesome
 - "God Puts you Through Troubled waters that he knows you're strong enough to swim through but your enemies cannot"
 
 ## Contact and Communication Preferences.
-For efficient communication, please feel free to reach out to me via email at my  I typically respond to emails within 24 hours during regular work hours, which are Monday through Friday, 9:00 AM to 5:00 PM, Eastern Standard Time. For urgent matters or time-sensitive issues, you can also reach me by phone at [your phone number]. Thank you for respecting these preferences, as they help maintain effective communication channels.
+For efficient communication, please feel free to reach out to me via email at my  I typically respond to emails within 24 hours during regular work hours, which are Monday through Friday, 9:00 AM to 5:00 PM, Eastern Standard Time. For urgent matters or time-sensitive issues, you can also reach me by phone at 302-592-8064. Thank you for respecting these preferences, as they help maintain effective communication channels.
```

Comment:
Might want to be careful about posting your phone number to the web. Sure you want to do that?